### PR TITLE
LNX-187 Show in GUI ranking of Player points

### PR DESCRIPTION
**Pull request by Krystian030**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -18,4 +18,4 @@ async def lifespan(app: FastAPI):
 app = FastAPI(lifespan=lifespan)
 app.add_middleware(GZipMiddleware, minimum_size=100)
 app.include_router(scene_router.router)
-app.include_router(server_router.router)
```
Comment (filip-szweda) [inline src/app.py:21]: https://github.com/group-project-gut/lynx-scene-host-python/pull/26/files#r1322057532

File: src/runtime.py [hunk 2]
```diff
@@ -38,7 +39,6 @@ def send(action: Action):
         logger.debug(f"Scene has been successfully received, starting to deserialize scene")
         scene = Scene.deserialize(scene_serialized)
         logger.debug(f"Scene has been successfully deserialized")
-
```
Comment (filip-szweda) [inline src/runtime.py]: https://github.com/group-project-gut/lynx-scene-host-python/pull/26/files#r1322057356

File: src/utils/helper_functions.py [hunk 3]
```diff
@@ -135,3 +135,13 @@ def handle_initial_player_connection(player: str) -> None:
...
         state.scene.generate_drop_area(player)
+
+
+async def calculate_player_rankings():
+    ranking = []
+    for player, points in state.scene.players_points.items():
+        sum_of_points = sum(points.values())
+        ranking.append((player, sum_of_points))
+
+    ranking.sort(key=lambda x: x[1], reverse=True)
```
Comment (filip-szweda) [inline src/utils/helper_functions.py:151]: Maybe `x` → `ranking_entry`

**Review by filip-szweda — APPROVED** [review]

**Review by EasyJB — APPROVED** [review]
LGood

File: src/app.py (hunk 1 above)
Comment (Krystian030) [reply src/app.py:21]: We are adding a new line here because it is good practice ;)

**Pull request merged**